lib/github/commands/man2html.xslt

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+  <!-- //
+    Based on the HTML output and default stylesheet of mandoc 1.14.4
+    YMMV with other versions of mandoc
+  // -->
+
+  <!-- Default: match anything and apply templates to it -->
+  <xsl:template match="node()|@*">
+    <xsl:copy>
+        <xsl:apply-templates select="node()|@*"/>
+    </xsl:copy>
+  </xsl:template>
+
+  <!-- Strip all the attributes off of these -->
+  <xsl:template match="h1">
+    <h1><xsl:apply-templates /></h1>
+  </xsl:template>
+
+  <!-- Turn these into paragraphs -->
+  <xsl:template match="div[@class='Pp']">
+    <p><xsl:apply-templates /></p>
```

Comment:
This replacement is WRONG, you can't do that.  In mandoc HTML output, paragraphs often contain flow content, but HTML only permits phrsing content in <p> elements.  So what you are doing here is introducing HTML syntax errors.

Reply:
@ischwarze You aren't wrong, but mistakes like these are, sadly, too damn prevalent throughout the rest of GitHub's infrastructure for yet one more violation to become noticed. *sigh*